### PR TITLE
Resolve patch/MOOD-44: user details component

### PR DESCRIPTION
**Pull request by katebrune**
added user details component, stories file, spec file, & added component to home page

### CODE REVIEW

File: ui/user-details/UserDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { UserInfo } from "../../types/UserInfo"
+
+interface UserDetailsProps {
+    user: UserInfo
+    small?: boolean
+}
+export const UserDetails: React.FunctionComponent<UserDetailsProps> = (props) => {
+    const { user, small } = props
+
+    if (small) {
```
Comment (lorenzoz23) [inline ui/user-details/UserDetails.tsx:14]: why not just conditionally set size inline instead of doubling the code with these two conditional returns?

Comment (lorenzoz23) [reply ui/user-details/UserDetails.tsx:14]: like just do size={small ? 'medium' : 'large'}

Comment (katebrune) [reply ui/user-details/UserDetails.tsx:14]: this one doesn't have the user name, its the small mobile one

File: ui/user-details/UserDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                        <User color="accent-1" size={"medium"} />
+                    </Avatar>
+                )}
+            </Box>
+        )
+    }
+    return (
+        <motion.div whileHover={{ scale: 1.1 }} whileTap={{ scale: 0.9 }}>
+            <Box direction="row" align="center" gap="small">
+                <Heading textAlign="center" margin="none" id="username-txt">
```
Comment (lorenzoz23) [inline ui/user-details/UserDetails.tsx:44]: yeah, i think we can just conditionally render the header with the one return statement.

so, {!small && (...)}. then, maybe we could test the small avatar rendering with a test that sees if the header is rendered or not depending on size

File: components/home/home.tsx [hunk 3]
```diff
@@ -115,31 +87,7 @@ export const Home: FunctionComponent<HomeProps> = (props) => {
...
-                            size={size !== "small" ? "large" : "medium"}
-                            onClick={() => window.open(props.user.profileUrl, "_blank")}
-                            title="click to open your spotify profile"
-                        >
-                            <User color="accent-1" size={size !== "small" ? "large" : "medium"} />
-                        </Avatar>
-                    )}
-                </Box>
-            )}
+            {size === "small" && <UserDetails user={user} small />}
```
Comment (lorenzoz23) [inline components/home/home.tsx:90]: yeah, i removed this in another branch to free up space, so you can just remove this line entirely and just render the avatar like you're doing on line 55

**Review by lorenzoz23 — CHANGES_REQUESTED** [review]

**Pull request merged**